docs/cody/use-cases/vsc-tutorial.mdx

```diff
@@ -0,0 +1,126 @@
+# Cody Tutorial
+
```

Comment:
<p className="subtitle">This tutorial gives you a quick walkthrough of leveraging Cody for autocompletion, chat, explaining code, and running commands in VS Code.</p>